field-testing/src/lib.rs

```diff
@@ -18,7 +18,7 @@ use num_bigint::BigUint;
 use p3_field::{
     ExtensionField, Field, PrimeCharacteristicRing, PrimeField32, PrimeField64, TwoAdicField,
     cyclic_subgroup_coset_known_order, cyclic_subgroup_known_order,
-    two_adic_coset_vanishing_polynomial, two_adic_subgroup_vanishing_polynomial,
```

Comment:
While we're at it, might as well remove `two_adic_subgroup_vanishing_polynomial` too, since it's no longer used anywhere.